screens/index.js

```diff
@@ -150,9 +150,9 @@ const Index = () => {
                   width:
                     Platform.OS === "web" ? window.width / 2 : window.width,
                 }}
-                onSubmitEditing={(e) => {
+                onSubmitEditing={({ nativeEvent: { text } }) => {
                   navigation.navigate("Search", {
-                    searchStr: e.target.value,
+                    searchStr: text,
```

Comment:
I remember this! We had to change from e.target.value to text because the object may not have .target! This depends on it being mobile or not, if I recall correctly